src/main/java/com/aviva/DataProcess/CarDataProcess.java

```diff
@@ -0,0 +1,58 @@
+package com.aviva.DataProcess;
+
+import com.aviva.DataAccess.SQLCarDataAccess;
+import com.aviva.Entities.Car;
+
+import java.sql.ResultSet;
+import java.sql.SQLException;
+import java.util.ArrayList;
+
+public class CarDataProcess implements CarDataProcessingInterface{
+    public static ArrayList<Car> getAllCars(){
```

Comment:
isn't this supposed to be "getNextCar"?

Reply:
Yeah - I just checked. This is not properly implementing the interface we defined...?

Reply:
you can keep this method as a temporary solution, but we should not violate the defined interface. Even if the getNextCar thing had a return null implementation.

Reply:
I talked to Maria about this and even she wasnt sure how to implement get next car so I have left it commented for now as 3 of us cant seem to figure out how itll be used or how it can be implemented.

Reply:
I think they are not doing getNextCar() since there is no way of marking an iteration on the database itself. We might want to change the interface in this case or investigate further into SQL and the database format they are using.